index.html

```diff
@@ -32,7 +32,149 @@ <h2 class="first-name software-dev">I'm a software developer</h2>
   <div class="arrow-flex display-flex">
     <span class="arrow">&or;</span>
 </div>
-</section>      
+</section>
+
+<!-- Recent Work Session -->
+  <h2 class="recent-work-align">My Recent</h2>
+  <h2 class="recent-work-align">Works</h2>
+<div class="dash-flex display-flex">
+<span class="dash">&#8212;</span>
+</div>
+<section class="container-recent">  
+  <div class="grid-project">
+    <div class="multi-post">
+     <h3 class="multi-text-color">Multi-Post Stories</h3>
+     <h3 class="multi-text-color">Gain+Glory</h3>
+     <ul class="language display-flex">
+     <li class="language-background">Ruby on rails</li>
+     <li class="language-background">css</li>
+     <li class="language-background" >JavaScript</li>
+     <li class="language-background">html</li>    
```

Comment:
### OPTIONAL

- [ ] Kindly add a margin of say 5px, to each li tag, so they don't look clustered on screens lesser than 425px, please do this for all li tags in each card👍🏿

Reply:
Thank you. Nevethless you should have known that my code included that and it does not cluster on screen less than 425px

  
![2022-04-25 (2)](https://user-images.githubusercontent.com/96915150/165098128-c2de551f-7349-46ab-a276-2e51a2f82094.png)
column-gap: 10px;
  margin-top: 8px;
  justify-content: center;
  padding: 0;
}